components/podio/sources/new-task-created/new-task-created.mjs

```diff
@@ -0,0 +1,53 @@
+import common from "../common/common-hook.mjs";
+
+export default {
+  key: "podio-new-task-created",
+  name: "New Task Created",
+  description: "Emit new events when a new task is created. [See the docs here](https://developers.podio.com/doc/hooks)",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  ...common,
+  props: {
+    ...common.props,
+    orgId: {
+      propDefinition: [
+        common.props.app,
+        "orgId",
+      ],
+    },
+    spaceId: {
+      propDefinition: [
+        common.props.app,
+        "spaceId",
+        (configuredProps) => ({
+          orgId: configuredProps.orgId,
+        }),
+      ],
+    },
+  },
+  methods: {
+    ...common.methods,
+    getMeta(event) {
+      return {
+        id: Date.now(),
+        ts: Date.now(),
+        summary: `New application created (ID:${event?.body?.task_id})`,
+      };
```

Comment:
Could we use the `task_id` as the id here? Also, there's a typo in the summary. "application" should say "task".
```suggestion
      return {
        id: event?.body?.task_id,
        ts: Date.now(),
        summary: `New task created (ID:${event?.body?.task_id})`,
      };
```